reasoner/controller/ConcludableController.java

```diff
@@ -72,12 +72,13 @@ private ConcludableController(Driver<CONTROLLER> driver, Concludable concludable
 
     @Override
     public void setUpUpstreamControllers() {
-        concludable.getApplicableRules(registry().conceptManager(), registry().logicManager())
-                .forEachRemaining(rule -> {
-                    Driver<? extends ConclusionController<INPUT, ?, ?>> controller = registerConclusionController(rule);
-                    conclusionControllers.put(rule.conclusion(), controller);
-                    conclusionUnifiers.put(rule.conclusion(), concludable.getUnifiers(rule).toSet());
-                });
+        iterate(registry().logicManager().applicableRules(concludable).entrySet()).forEachRemaining( ruleAndUnifiers -> {
+                Rule rule = ruleAndUnifiers.getKey();
+                Set<Unifier> unifiers = ruleAndUnifiers.getValue();
+                Driver<? extends ConclusionController<INPUT, ?, ?>> controller = registerConclusionController(rule);
+                conclusionControllers.put(rule.conclusion(), controller);
+                conclusionUnifiers.put(rule.conclusion(), unifiers);
```

Comment:
this will change back to the older style if we go back to separate APIs for `rules` and `unifiers`